protocol/x/feetiers/keeper/params.go

```diff
@@ -27,6 +30,40 @@ func (k Keeper) SetPerpetualFeeParams(
 		return err
 	}
 
+	lowestMakerFee := int32(math.MaxInt32)
+	lowestTakerFee := int32(math.MaxInt32)
+	for _, tier := range params.Tiers {
```

Comment:
Instead of duplicating logic let's make `GetLowestMaker/TakerFee` take in the fee tiers and call them here